about.html

```diff
@@ -0,0 +1,126 @@
+          </ul>
+        </nav>
+       </div>
+      </div>
+    </div>
+    <section id="hero-section">
+      <div class="about-content">
+        <h2 class="about-subtitle sub-title">NGC Dev Congress</h2>
+        <h1 class="about-heading">THE WORLD'S LEADING DEVELOPER EVENT</h1>
+        <p class="about-desc">Animi esse, sapiente cum odio nulla neque dolores iste totam? uasi molestias deserunt scipit nam, sapiente cum odio nulla neque dolores iste totam? ea dolorum, dolor quidem voluptatibus modi magni molestias vitae at. Id mollitia repellendus consectetur!</p>
+        <p class="about-small-desc">Lorem ipsum dolor sit amet consectetur adipisicing elit. Numquam est ut sunt maiores.</p>
+        <a href="#" class="about-email">sekodameh@gmail.com</a>
+      </div>
+    </section>
+  </header>
+
+  <main>
+    <section id="about-event-section">
+      <h3 class="about-event-title">Next Generation Coders World's Developer Event</h3>
+      <div class="title-line-container">
+        <p class="title-line"></p>
+        <p class="title-line"></p>
+      </div>
+      <p class="about-event-desc">Lorem ipsum, dolor sit amet consectetur adipisicing elit. Placeat nulla id voluptatem mollitia, fugiat sapiente.</p>
+      <div class="about-logo">
+        <img src="img/nextgencodez_logo.png" alt="">
+      </div>
+    </section>
+    <hr>
+    <section id="about-bootchamp-section">
+      <h3 class="past-event-title">See the past NGC World's Developer Event</h3>
+      <div class="title-line-container">
+        <p class="title-line"></p>
+        <p class="title-line"></p>
+      </div>
+      <p class="past-event-desc">Lorem ipsum, dolor sit amet elit. Placeat nulla id voluptatem mollitia, fugiat sapiente.</p>
+      <div class="bootchamps">
+        <div class="bootchamp-one">
+            <p class="event-year">2020</p>
+            <p class="event-location">Lorem ipsum dolor sit amet.</p>
+        </div>
+        <div class="bootchamp-two">
+            <p class="event-year">2020</p>
+            <p class="event-location">Lorem ipsum dolor sit amet.</p>
+        </div>
+        
+      </div>  
+    </section>
+
+    <section class="partners-section">
+      <h3 class="section-title">Partners</h3>
+      <div class="title-line-container">
+        <p class="title-line"></p>
+        <p class="title-line"></p>
+      </div>
+      <div class="partners-container">
+        <img src="img/google.png" alt="Google">
+        <img src="img/airbnb.png" alt="Airbnb">
+        <img src="img/mozila.png" alt="Mozilla">
+        <img src="img/daum.png" alt="Daum">
+        <img src="img/naver.png" alt="Naver">
+      </div>
+    </section>
+
+    <footer>
+      <div class="footer-container">
+        <div class="logo-container">
+          <img src="img/nextgencodez-logo.png" alt="">
+        </div>
+        <div class="footer-content">
+          <p>2022 creative commons Ghana. All rights Reserved</p>
+        </div>
+      </div>
+    </footer>
```

Comment:
- [ ] Please make sure the footer of the about page has a dark background to match the design guidelines. You are expected to stick to design guidelines as much as possible.
### Your design
![1](https://user-images.githubusercontent.com/94551484/196765860-04062dfb-a69b-463f-a8ce-ba30b7aa702f.png)

### Design guidelines
![f1](https://user-images.githubusercontent.com/94551484/196765898-e8af703c-0524-4830-b1ec-0e9e0e7ce462.png)